editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -537,23 +539,24 @@ export function useSelectAndHover(
 } {
   const modeType = useEditorState((store) => store.editor.mode.type, 'useSelectAndHover mode')
   const selectModeCallbacks = useSelectModeSelectAndHover(
```

Comment:
hmmm if this now covers select, select-lite and live as well, I think we shouldn't be calling this `useSelectModeSelectAndHover` anymore

```suggestion
  const selectAndHoverCallbacks = useSelectAndHover(
```

Reply:
Yep but that name is already taken lower down in the file! Will change it though

Reply:
I've renamed to `useSelectOrLiveModeSelectAndHover` (it was already covering live mode, and select-lite is planned to be binned in the near-ish term)

Reply:
thanks :)